### ISSUE TITLE
Error while loading plugin

### ISSUE THREAD

**Issue by konart** [AlexvZyl/nordic.nvim#14]
```
E5108: Error executing lua ...nordic.nvim/lua/nordic/groups/integrations/telescope.lua:83: attempt to index field 'telescope'
(a nil value)
```

I do have config block (copy-pasted from readme)
```
	use({
		"AlexvZyl/nordic.nvim",
		config = function()
			require("nordic").setup({
				telescope = {
					style = "flat",
				},
			})
		end,
	})
```

**Comment by AlexvZyl** [comment]
I can't reproduce this.  Are you on latest?

**Comment by AlexvZyl** [comment]
My bad.  This happens when you do not call `setup`.  Will fix now.

**Comment by AlexvZyl** [comment]
Fixed in 9abc2ba7fdf2cd192cd79525ea9f56d284a2cce8.

**Comment by konart** [comment]
telescope.style can also have default value.
